src/db/spanner/models.rs

```diff
@@ -243,11 +246,13 @@ impl SpannerDb {
             .sql(
                 "SELECT CURRENT_TIMESTAMP(), last_modified
                    FROM user_collections
-                  WHERE userid = @userid
+                  WHERE fxa_kid = @fxa_kid
```

Comment:
This is an annoying ask but can we keep the order of columns intact everywhere (user_id, kid)? w/ kids being a piece of metadata on users

Reply:
ok, i've tried to restore the order of columns (userid first, then fxa_kid, etc. to match the indexes) also tried to make sure that the parameter lists match up as well, since we have two values that are only different by one letter.